riptable/rt_str.py

```diff
@@ -12,13 +15,32 @@
 import re
 import numpy as np
 import numba as nb
+from numba.core.dispatcher import Dispatcher
 
 from .config import get_global_settings
 from .rt_fastarray import FastArray
 from .rt_numpy import empty_like, empty, where, ones, zeros
 from .rt_enum import TypeRegister
 
 
+# Partially-specialize the numba.njit decorator to simplify it's use in the FAString class below.
+_njit_serial = partial(nb.njit, parallel=False, cache=get_global_settings().enable_numba_cache, nogil=True)
+_njit_par = partial(nb.njit, parallel=True, cache=get_global_settings().enable_numba_cache, nogil=True)
+
+class FAStrDispatchPair(NamedTuple):
+    """A pair of numba-based functions; one compiled for serial execution and the other for parallel execution."""
+
+    serial: Dispatcher
+    parallel: Dispatcher
+
+    @staticmethod
+    def create(py_func: callable) -> 'FAStrDispatchPair':
+        func_serial = _njit_serial(py_func)
+        func_par = _njit_par(py_func)
+        print(f"Type of func_serial: {type(func_serial).__qualname__}")

```

Comment:
Oops, I left a little bit of debugging code in here too. I'll remove this line + fix the spelling error above.